includes/admin/class-wc-stripe-settings-controller.php

```diff
@@ -264,6 +264,7 @@ public function admin_scripts( $hook_suffix ) {
 			'are_apms_deprecated'          => WC_Stripe_Feature_Flags::are_apms_deprecated(),
 			'is_amazon_pay_available'      => WC_Stripe_Feature_Flags::is_amazon_pay_available(),
 			'is_oc_available'              => WC_Stripe_Feature_Flags::is_oc_available(),
+			'is_oc_enabled'                => WC_Stripe_Feature_Flags::is_oc_available() && 'yes' === $this->get_gateway()->get_option( 'optimized_checkout_element' ),
```

Comment:
We can use `is_oc_enabled()` from the gateway (if exists 🤷🏼):
```suggestion
			'is_oc_enabled'                => method_exists( $this->get_gateway(), 'is_oc_enabled' ) && $this->get_gateway()->is_oc_enabled(),
```

Reply:
Thanks for the suggestion, but with the changes in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4453/commits/4f5a7bd3fd59dea7ba8621b2af4357c353454488 that's not needed anymore.